index.html

```diff
@@ -120,81 +126,82 @@
   <section id="abstract">
     <p>
       This specification defines a <a href="#profiling-mechanism">Profiling
-      Mechanism</a> and a set of <a href="#dfn-profile">Profiles</a>
+        Mechanism</a> and a set of <a>Profiles</a>
       which enable <a href="#out-of-the-box-interoperability">out-of-the-box
-      interoperability</a> between
-      <a href="#dfn-thing">Web Things</a> and their
-      <a href="#dfn-consumer">Consumers</a> on the
+        interoperability</a> between
+      <a>Web Things</a> and their
+      <a>Consumers</a> on the
       <a href="https://www.w3.org/wot">Web of Things</a>.
     </p>
     <p>
-      Being out-of-the-box interoperable means that any Consumer which conforms
-      with a given Profile can interact with any Thing which conforms with the
-      same profile, without additional customization.
+      Being out-of-the-box interoperable means that any <a>Consumer</a> which conforms
+      with a given <a>Profile</a> can interact with any <a>Thing</a> which conforms with the
+      same <a>Profile</a>, without additional customization.
     </p>
     <p>
       A Profile is a technical specification which provides a set of assertions
-      to which conformant Consumers and Things must conform.
+      to which conformant <a>Consumers</a> and <a>Things</a> must conform.
     </p>
     <p>
       The Profiling Mechanism provides a means to denote that a given Thing
-      conforms to one or more Profiles, by referring to the identifiers of those
-      Profiles in the <code>profile</code> member of its
-      <a href="#dfn-thing-description">Thing Description</a>.
+      conforms to one or more <a>Profiles</a>, by referring to the identifiers of those
+      <a>Profiles</a> in the <code>profile</code> member of its
+      <a>Thing Description</a>.
     </p>
 
     <p>
       The specification defines three profiles:
-      <ul>
-        <li>The <a href="#http-basic-profile">HTTP Basic
+    <ul>
+      <li>The <a href="#http-basic-profile">HTTP Basic
           Profile</a> defines a protocol binding for reading and writing
-          properties and invoking, querying and cancelling actions using the
-          HTTP protocol.
-        </li>
-        <li>
-          The <a href=#sec-http-sse-profile>HTTP SSE
+        properties and invoking, querying and cancelling actions using the
+        HTTP protocol.
+      </li>
+      <li>
+        The <a href=#http-sse-profile>HTTP SSE
```

Comment:
The paragraph below has minor editorial changes and they are fine as well to me.